src/site/content/en/blog/profiling-web-audio-apps-in-chrome/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Profiling Web Audio in Chrome
+subhead: Learn how to profile the performance of Web Audio apps in Chrome using chrome://tracing and the WebAudio panel in DevTools.
+description: Learn how to profile the performance of Web Audio apps in Chrome using chrome://tracing and the WebAudio panel in DevTools.
+date: 2020-04-30
+updated: 2020-04-30
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - media
+  - webaudio
+authors:
+  - hongchanchoi
+---
+
+You reached this documentation probably because you're developing on an app
+that uses the Web Audio API and experienced some unexpected glitches such as
+popping noises from the output. You might already be involved in a
+[crbug.com](https://crbug.com) discussion and a Chrome engineer asked you to
+upload "tracing data". This document shows how to obtain the data so you can
+help engineers triage, and eventually fix the issue.
+
+## Web Audio profiling tools
+
+There are two tools that will help you when profiling Web Audio,
+`chrome://tracing` and the WebAudio panel in Chrome DevTools.
```

Comment:
```suggestion
`chrome://tracing` and the **WebAudio** tab in Chrome DevTools.
```
* Formatted bold per style guide: https://developers.google.com/style/ui-elements#formatting
* Renamed to `tab` to keep it consistent with the official Chrome DevTools documentation (which I wrote most of)